app/controllers/organizations_controller.rb

```diff
@@ -1,4 +1,4 @@
-class OrganizationsController < ApplicationController
+class OrganizationsController < Organizations::BaseController
```

Comment:
the show action on this controller serves both the public (the organization page anyone can see) and the Organization page for members. 

I think we can leave this alone, as the controller/views handle these variants currently, what are your thoughts?

Reply:
For the purposes of this PR, I think it makes sense to use `skip_before_action` to allow everyone to view those pages.

The proposed change here (swapping out the base) does have bigger implications. Do you anticipate these pages/actions changing for different reasons than the rest of the organization-related work? If so, keeping the former inheritance seems 👍